eng/scripts/automation/Invoke-CadlDataPlaneGenerateSDKPackage.ps1

```diff
@@ -52,8 +52,37 @@ param (
   [string]$specRoot = "",
   [string]$repo="",
   [string]$commit="",
-  [string]$additionalSubDirectories="" #additional directories needed, separated by semicolon if more than one
+  [string]$additionalSubDirectories="", #additional directories needed, separated by semicolon if more than one
+  [switch]$help
 )
+if ($help) {
+  Write-Host("Usage:
+Invoke-CadlDataPlaneGenerateSDKPackage.ps1 -service <servicename> -namespace Azure.<group>.<service> -sdkPath <sdkrepoRootPath> -cadlRelativeFolder <relativeCadlProjectFolderPath> [-commit <commitId>] [-repo <specRepo>] [-specRoot <specRepoRootPath>] [-additionalSubDirectories <relativeFolders>]
+
+Options:
+-service  [Required] takes Azure client service directory name in sdk repo. e.g. AnomalyDetector
+-namespace  [Required] takes the namespace of the Azure client library.
```

Comment:
I wonder if we should just take the project folder name since it should match and will eliminate the need for service directory since it will always be a sub-directory?

Reply:
We will use the sdk folder path instead of `service` and `namespace` parameters for this script.